src/ports/postgres/modules/utilities/sessionize.py_in

```diff
@@ -35,41 +36,57 @@ def sessionize(schema_madlib, source_table, output_table, partition_expr,
         @param source_table: str, Name of the input table/view
         @param output_table: str, Name of the table to store result
         @param partition_expr: str, Expression to partition (group) the input data
-        @param time_stamp: str, Column name with time used for sessionization calculation
+        @param time_stamp: str, The time stamp column name that is used for sessionization calculation
         @param max_time: interval, Delta time between subsequent events to define a session
-
+        @param output_cols: str, a valid postgres SELECT expression
+        @param create_view: boolean, indicates if the output is a view or a table with name specified by output_table (default TRUE)
+                        TRUE - create view
+                        FALSE - materialize results into a table
     """
     with MinWarning("error"):
         _validate(source_table, output_table, partition_expr, time_stamp, max_time)
+        table_or_view = 'VIEW' if create_view or create_view is None else 'TABLE'
+
+        output_cols = '*' if output_cols is None else output_cols
+        # If the output_cols has '*' as one of the elements, expand it to include all columns in the source table. The following list
```

Comment:
This block needs to be wrapped into 72 characters and appropriate line space needs to be added to improve readability.